### PR TITLE
Full Feature Settings Page - Scrolling Bug

### PR DESCRIPTION
**Pull request by StioStudio**
Before: 
image
Now: 
image

### CODE REVIEW

File: extras/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       </div>
     </div>
     <div class="section1">
-      <hr style="border: none; height: 1.5rem" />
+      <!-- <hr style="border: none; height: 1.5rem" /> -->
```
Comment (MaterArc) [inline extras/index.html:29]: If you've throughly tested it across multiple browsers and it works to fix the bug. You don't need to comment any of these out, and can just go ahead and delete them.

Comment (MaterArc) [reply extras/index.html:29]: Any, being the ones that you commented out across all files of course.

Comment (StioStudio) [reply extras/index.html:29]: Oh yeah. I commented them because that was the way I deleted them. I can take them away

Comment (MaterArc) [reply extras/index.html]: All good.

**Comment by StioStudio** [comment]
Is this gonna be added to the next update?

**Review by rgantzos — APPROVED** [review]

**Pull request merged**